packages/perseus/src/widgets/interactive-graphs/mafs-graph.tsx

```diff
@@ -91,7 +106,7 @@ export const MafsGraph = React.forwardRef<
     const [xSnap, ySnap] = props.snapStep;
     useEffect(() => {
         dispatch(changeSnapStep([xSnap, ySnap]));
-    }, [xSnap, ySnap]);
+    }, [dispatch, xSnap, ySnap]);
```

Comment:
`dispatch` is required by the linter, but IIRC `dispatch` is guaranteed to not change.